CITATION.bib

```diff
@@ -1,11 +1,11 @@
-@article{julia2017,
-  title={Julia: A fresh approach to numerical computing},
-  author={Bezanson, Jeff and Edelman, Alan and Karpinski, Stefan and Shah, Viral B},
-  journal={SIAM {R}eview},
-  volume={59},
-  number={1},
-  pages={65--98},
-  year={2017},
-  publisher={SIAM},
-  url={https://doi.org/10.1137/141000671}
+@article{Julia.jl,
+    title={Julia: A fresh approach to numerical computing},
+    author={Bezanson, Jeff and Edelman, Alan and Karpinski, Stefan and Shah, Viral B},
```

Comment:
Is this really how names are supposed to be listed in bibtex entries? It looks so bizarre. It looks like this paper is the work of
* Bezanson
* Jeff and Edelman
* Alan and Karpinski
* Stefan and Shah
* Viral B

It's like our names got put in a blender and fished out.

Reply:
> Is this really how names are supposed to be listed in bibtex entries

Yes, the "and" keyword separates authors, the comma separates the family name(s) from the given names of each author.

Reply:
Yup. That is right, and bibtex knows how to deal with it also. This bibtex entry is from SIAM - not something I wrote.